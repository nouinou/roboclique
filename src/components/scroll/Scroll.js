const Scroll = (props) => {
	return <div className="h-content overflow-x-auto">{props.children}</div>;
};

export default Scroll;
