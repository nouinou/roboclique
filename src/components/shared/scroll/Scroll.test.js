import { shallow } from 'enzyme';
import Scroll from './Scroll';

describe('Scroll Component', () => {
	it('should render Scroll', () => {
		expect(shallow(<Scroll />)).toMatchSnapshot();
	});
});
