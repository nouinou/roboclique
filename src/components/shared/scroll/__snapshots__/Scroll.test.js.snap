// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll Component should render Scroll 1`] = `
<div
  className="h-content overflow-x-auto"
/>
`;
